x-pack/plugin/sql/qa/server/src/main/resources/setup_test_emp.sql

```diff
@@ -9,4 +9,17 @@ CREATE TABLE "test_emp" (
                     "last_name" VARCHAR(50),
                     "salary" INT
                    )
-   AS SELECT * FROM CSVREAD('classpath:/employees.csv');
\ No newline at end of file
+   AS SELECT * FROM CSVREAD('classpath:/employees.csv');
+   
+DROP TABLE IF EXISTS "logs";
```

Comment:
To Andrei's point, isn't logs.csv used already and if not can you track down why we add it in the first place? Additionally instead of adding a new data file (assuming it is not used) you can emulate the null using a CASE function for bonus points.

Reply:
The `logs.csv` was only used before in `*.csv-spec` tests and loaded by the `DataLoader` by this method:

https://github.com/elastic/elasticsearch/blob/c874e6cdd3e051ce599b50c18642de038b84105f/x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/DataLoader.java#L65

, but it was not loaded during the `*.sql-spec` tests into H2.

`logs` is used in the `pivot.csv-spec` and `ip.csv-spec`. 

Regarding CASE: Originally was thinking about using this trick, but the CASE only works for the `stats` and `extended_stats` aggregations, and does not work for the `matrix_stats` aggregations, hence I decided to go with just a field instead of emulating null with CASE.

```
SELECT client_port
  , SKEWNESS(CASE WHEN bytes_in < 30 THEN NULL ELSE bytes_in END)
FROM logs
GROUP BY client_port

Found 1 problem
line 2:14: [SKEWNESS()] cannot be used on top of operators or scalars
```